main/midnight/src/system/moonring.cpp

```diff
@@ -576,6 +576,7 @@ void moonring::initialise( progressmonitor* monitor )
     tme::variables::sv_cheat_armies_noblock = true;
     tme::variables::sv_cheat_nasties_noblock = true;
     tme::variables::sv_cheat_movement_free = true ;
+    tme::variables::sv_cheat_always_win_fight = true;
```

Comment:
Would you be ok if his cheat was added to 'default' cheat list? Really handy... ;)

Reply:
Yes, this is fine. Also check out configmanager.

you can put a file in the same folder as settings are stored Can't remember where that is on windows. 

```
<?xml version="1.0" encoding="ISO-8859-1" ?>
<main>
  <variables>
    <var id="cheat_always_win_fight" value="false" />

    <var id="skip_dedication" value="true" /> 
  </variables>
</main>
```